db/migration/pre-migrations-schema.sql

```diff
@@ -0,0 +1,447 @@
+  `published_at` datetime DEFAULT NULL,
+  `updated_at` datetime NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_as_cs;
+
+CREATE TABLE `namespaces` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `description` varchar(255) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `isArchived` tinyint(1) NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `namespaces_name_uq` (`name`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_as_cs;
+
+CREATE TABLE `entities` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `code` varchar(255) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `name` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `validated` tinyint(1) NOT NULL,
+  `createdAt` datetime NOT NULL,
+  `updatedAt` datetime NOT NULL,
+  `displayName` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `name` (`name`),
+  UNIQUE KEY `code` (`code`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_as_cs;
+
+CREATE TABLE `importer_additionalcountryinfo` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `country_code` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `country_name` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `country_wb_region` varchar(255) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `country_wb_income_group` varchar(255) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `country_special_notes` longtext COLLATE utf8mb4_0900_as_cs,
+  `country_latest_census` varchar(255) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `country_latest_survey` varchar(255) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `country_recent_income_source` varchar(255) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `dataset` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_as_cs;
+
+CREATE TABLE `importer_importhistory` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `import_type` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `import_time` datetime NOT NULL,
+  `import_notes` longtext COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `import_state` longtext COLLATE utf8mb4_0900_as_cs NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_as_cs;
+
+
+CREATE TABLE `datasets` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(512) COLLATE utf8mb4_0900_as_cs DEFAULT NULL,
+  `description` longtext COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `createdAt` datetime NOT NULL,
+  `updatedAt` datetime NOT NULL,
+  `namespace` varchar(255) COLLATE utf8mb4_0900_as_cs NOT NULL,
+  `isPrivate` tinyint(1) NOT NULL DEFAULT '0',
+  `createdByUserId` int NOT NULL,
+  `metadataEditedAt` datetime NOT NULL,
+  `metadataEditedByUserId` int NOT NULL,
+  `dataEditedAt` datetime NOT NULL,
+  `dataEditedByUserId` int NOT NULL,
+  -- `nonRedistributable` tinyint(1) NOT NULL DEFAULT '0',
```

Comment:
Oof this file must have taken a while 😓 

What's up with the commented out statements throughout?